src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -1853,6 +1852,56 @@ bool AdaptDispatch::SetColorTableEntry(const size_t tableIndex,
     return fSuccess;
 }
 
+// Method Description:
+// - Sets the default foreground color to a new value
+// Arguments:
+// - dwColor: The new RGB color value to use, as a COLORREF, format 0x00BBGGRR.
+// Return Value:
+// True if handled successfully. False othewise.

```

Comment:
There's a typo here in `otherwise` and below and probably from wherever this was copied from.

Reply:
Yeah, I noticed that same typo elsewhere just now, I'll fix this one too.

Reply:
LOL It's spelled that way 21 times this file. Fixed.